### DISCUSSION TITLE
My Uni does not provide an email adress

### DISCUSSION THREAD

**Discussion by Gwendolice** [community/community#148071] [GitHub Education]
### Hi everyone,

Hi! The title is self explanatory, My Univeristy does not provide an email adress, but they do provide digital papers that says I study there, how can I get approved for students plan if the website says it only allows university-provide email adresses? 

The Uni's name is "Guillermo Brown National University" and the domain is unab.edu.ar

Ty!

**Comment by Lut-Lat-Aung** [comment]
Unfortunately, if your university no longer provides you with an official email address, obtaining the GitHub Student Developer Pack might be a bit more challenging, as GitHub typically requires a university-issued email for verification.

You can try other things

- Maybe, try contacting the Github support with your personal email to approve your student developer pack if you can show university transcript and student ID.

I hope you find my answer helpful.
